server/db/models/category.js

```diff
@@ -1,11 +1,9 @@
-//This is the category model.
-
 'use strict';
 var Sequelize = require('sequelize');
-
+var Product = require('./product');
```

Comment:
No need to require `./product` into this file given that you're not using it here.